examples/generate_txs.rs

```diff
@@ -27,10 +28,11 @@ async fn main() -> Result<(), Reason> {
         sechash_map.insert(0xff, HEXLOWER.encode(&tx.raw_header_hash().0));
         ledger_vector.name = format!("{}_{}", i, ledger_vector.name);
         test_vectors.push(ledger_vector.clone());
-        debug_vectors.push((ledger_vector, tx, tx_data, sechash_map));
+        debug_vectors.push((ledger_vector, tx.clone(), tx_data, sechash_map));
+        serialized_txs.push(HEXLOWER.encode(tx.to_bytes().as_ref()));
     }
     let args: Vec<_> = std::env::args().collect();
-    if args.len() < 3 {
+    if args.len() < 4 {
```

Comment:
```suggestion
    if args.len() < 3 {
```
let's make the new last arg optional